collection/collection.go

```diff
@@ -70,3 +70,8 @@ type Map interface {
 	// Remove deletes the key from the CollectionMap
 	Remove(key string)
 }
+
+// Noop is a dummy Collection.
```

Comment:
We don't need a public interface for this the methods can just return Collection